Fix no_std build (#368)
crossbeam-rs/crossbeam: crossbeam-epoch/README.md

shared objects until no pointers to them can exist.

Everything in this crate except the global GC can be used in `no_std` environments that implement

> `alloc`. It needs to enable the `alloc` feature of this crate to use this crate, in the `no_std` environments.

Reviewer: Perhaps we could rephrase this as:

```suggestion
`alloc`. The `alloc` feature of this crate needs to be enabled in `no_std` environments.
```

Author: Yeah.
